subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2512,10 +2555,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_SUBCMD_SET_END
 );
 
-SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
-	SHELL_CMD_ARG(version, NULL, "Print Wi-Fi Driver and Firmware versions\n",
-		  cmd_wifi_version,
-		  1, 0),
+SHELL_STATIC_SUBCMD_SET_CREATE(
+	wifi_commands,
+	SHELL_CMD_ARG(version, NULL, "Print Wi-Fi Driver and Firmware versions\n", cmd_wifi_version,
+		      1, 0),
```

Comment:
whitespace change, please remove or move to a separate commit.

Reply:
Updated as comment